### PR TITLE
chore: Updating versions to include current version packages in distrib

### PR DESCRIPTION
**Pull request by MMaiero**
> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped

**Screenshots:** If applicable, add screenshots to help explain your solution

**Manual Tests**: Optional description of the tests performed to check correct functioning of changes, useful for an efficient review

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

File: target-platform/config/kura.target-platform.build.properties [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+org.eclipse.soda.dk.comm.version=1.8.0-SNAPSHOT
+org.usb4java.version=1.8.0-SNAPSHOT
+usb4java-javax.version=1.8.0-SNAPSHOT
+com.gwt.user.version=1.7.0-SNAPSHOT
+jdk.dio.version=1.6.0-SNAPSHOT
+org.eclipse.kura.camel.sun.misc.version=1.6.0-SNAPSHOT
+org.eclipse.kura.sun.misc.version=1.6.0-SNAPSHOT
+org.eclipse.kura.jetty.customizer.version=1.6.0-SNAPSHOT
 
 org.moka7.version=1.5.0
```
Comment (nicolatimeus) [inline target-platform/config/kura.target-platform.build.properties]:
```suggestion
org.moka7.version=1.6.0-SNAPSHOT
```

**Comment by nicolatimeus** [comment]
`libsocket-can-osgi.version` and `log4j2-api-config` need update as well

**Comment by MMaiero** [comment]
@nicolatimeus changes implemented

**Comment by MMaiero** [comment]
Forcing merge. Currently the Eclipse Jenkins instance is not working properly. We are going to merge the PRs and then perform a full build on Eclipse release-5.6.0 maintenance branch with, eventually, a following PR to fix remaining issues

**Pull request merged**